### PR TITLE
Add a tooltip component for template or external apps (#4518)

### PR DESCRIPTION
**Pull request by vlo-rte**
- In release note :
  -  In chapter : Features
  -  Text : #4518 : Add a tooltip component for template or external apps

### CODE REVIEW

File: src/test/externalWebAppExample/index.html [hunk 1]
```diff
@@ -185,6 +190,11 @@ <h3>Darker background</h3>
                 <button type="button" class="opfab-btn" disabled>DISABLED</button>
                 <button type="button" class="opfab-btn-cancel">CANCEL</button>
             </div>
+            <br />
+            <p>TOOLTIP</p>
+            <div>
+                <span data-text="Here is an example of tooltip" class="opfab-tooltip">Some tooltip text</span>
```
Comment (freddidierRTE) [inline src/test/externalWebAppExample/index.html:196]: replace `data-text `by `opfab-tooltip-text `

File: src/test/externalWebAppExample/index.html [hunk 2]
```diff
@@ -111,6 +111,11 @@ <h3>Lighter background</h3>
...
```
Comment (freddidierRTE) [inline src/test/externalWebAppExample/index.html:117]: replace `data-text `by `opfab-tooltip-text `

File: ui/main/src/shared/css/opfab-application.css [hunk 3]
```diff
@@ -619,3 +647,180 @@ input:-webkit-autofill {
...
 /* END BUTTON STYLE */
+
+.opfab-tooltip {
+    position: relative; /* making the .tooltip span a container for the tooltip text */
+    font-weight: normal;
+    font-style: normal;
+}
+
+.opfab-tooltip:before {
+    content: attr(data-text); /* here's the magic */
```
Comment (freddidierRTE) [inline ui/main/src/shared/css/opfab-application.css:658]: replace `data-text `by `opfab-tooltip-text `

File: src/test/cypress/cypress/integration/Timeline.spec.js [hunk 4]
```diff
@@ -859,11 +859,11 @@ describe('Time line moves', function () {
     }
 
     function lockTimeLine() {
-        cy.get("#opfab-timeline-lock").click();
+        cy.get("#opfab-timeline-lock").click({force: true});
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/Timeline.spec.js:862]: add only if necessary  ?

**Comment by freddidierRTE** [comment]
Add a chapter in documentation regarding the tooltip : https://opfab.github.io/documentation/archives/SNAPSHOT/docs/single_page_doc.html#opfab_template_style

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: src/test/cypress/cypress/integration/Timeline.spec.js (hunk 4 above)
Comment (vlo-rte) [reply src/test/cypress/cypress/integration/Timeline.spec.js:862]: Done.

File: ui/main/src/shared/css/opfab-application.css (hunk 3 above)
Comment (vlo-rte) [reply ui/main/src/shared/css/opfab-application.css:658]: Done.

File: src/test/externalWebAppExample/index.html (hunk 2 above)
Comment (vlo-rte) [reply src/test/externalWebAppExample/index.html:117]: Done.

File: src/test/externalWebAppExample/index.html (hunk 1 above)
Comment (vlo-rte) [reply src/test/externalWebAppExample/index.html:196]: Done.

File: src/test/resources/bundles/defaultProcess_V1/template/template.handlebars [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
+<!-- SPDX-License-Identifier: MPL-2.0                                      -->
+<!-- This file is part of the OperatorFabric project.                      -->
 
 <h4> Hello {{userContext.login}}, you received the following message </h4>
 
 <br/>
 
 <div class="template-style">
-{{card.data.message}}
+<span data-text="After reading the message, please acknowledge the card." class="opfab-tooltip">{{card.data.message}}</span>
```
Comment (freddidierRTE) [inline src/test/resources/bundles/defaultProcess_V1/template/template.handlebars:14]: opfab-tooltip-text

File: src/test/cypress/cypress/integration/Timeline.spec.js [hunk 6]
```diff
@@ -863,7 +863,7 @@ describe('Time line moves', function () {
     }
 
     function unlockTimeLine() {
-        cy.get("#opfab-timeline-unlock").click();
+        cy.get("#opfab-timeline-unlock").click({force: true});
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/Timeline.spec.js:866]: to remove

File: src/test/resources/bundles/defaultProcess_V1/template/template.handlebars (hunk 5 above)
Comment (vlo-rte) [reply src/test/resources/bundles/defaultProcess_V1/template/template.handlebars:14]: Done.

File: src/test/cypress/cypress/integration/Timeline.spec.js (hunk 6 above)
Comment (vlo-rte) [reply src/test/cypress/cypress/integration/Timeline.spec.js]: Done.

**Pull request merged**